resources/views/admin/timelog/index.blade.php

```diff
@@ -0,0 +1,49 @@
+@extends('layout')
+@section('content')
+<div class="container">
+    <div class="row justify-content-end pt-2">
+        <form method="GET" action="{{ route('manage.timelogs.create') }}">
+            <button class="btn btn-secondary">Create Time Log</button>
+        </form>
+    </div>
+    <div class="row trans-table">
+        <table class="table table-hover">
+            <thead class="thead-dark">
+                <tr>
+                    <th scope="col">Day</th>
+                    <th scope="col">Check In</th>
+                    <th scope="col">Check Out</th>
+                    <th scope="col">Name Staff</th>
+                    <th scope="col">User ID</th>
+                    <th scope="col">Action</th>
+                </tr>
+            </thead>
+            <tbody>
+                @foreach ($timeLogs as $timeLog)
+                <tr>
+                    <td>{{ $timeLog->day }}</td>
+                    <td>{{ $timeLog->check_in }}</td>
+                    <td>{{ $timeLog->check_out }}</td>
+                    <td>
+                        <a href="{{ route('manage.timelogs.timelog_user', $timeLog->user_id) }}">
+                            {{ $timeLog->user->name }}
```

Comment:
link to user profile here, it does not make sense for linking to `manage.timelogs.timelog_user`

Reply:
I have fixed this